sdk/storage/storage-blob/src/sas/AccountSASPermissions.ts

```diff
@@ -54,6 +54,9 @@ export class AccountSASPermissions {
         case "f":
           accountSASPermissions.filter = true;
           break;
+        case "i":
+          accountSASPermissions.setImmutabilityPolicy = true;
```

Comment:
>setImmutabilityPolicy [](http://example.com/codeflow?start=32&length=21)

see if version check is needed

Reply:
added

Reply:
'i' permission require minimum version of "2020-08-04". Added checking.